examples/blender/blender.py

```diff
@@ -73,9 +73,12 @@ async def worker(ctx: WorkContext, tasks):
 
     # iterator over the frame indices that we want to render
     frames: range = range(0, 60, 10)
-    # TODO make this dynamic, e.g. depending on the size of files to transfer
+    # TODO make this more dynamic, e.g. depending on the size of files to transfer
     # worst-case time overhead for initialization, e.g. negotiation, file transfer etc.
-    init_overhead: timedelta = timedelta(minutes=3)
+    init_overhead, min_timeout, max_timeout = 3, 7, 28
```

Comment:
Why `7` and `28`? If we're going to explain why we have those values by referring to the [5 min, 30 min] range used by the providers then IMO it's better to make it `6` and `30`, respectively.

Reply:
Also, since we know that the bounds are not configurable and are going to be 5 min and 30 min for all providers, maybe we should move this clipping operation to `yapapi` itself? And emit a message if the timeout set by the user is outside of the range of accepted timeout values?